h5py/tests/old/test_h5d_direct_chunk_write.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+import h5py
+import numpy
+
+from .common import ut, TestCase
+
+
+@ut.skipUnless(h5py.version.hdf5_version_tuple >= (1, 8, 11), 'Direct Chunk Writing requires HDF5 >= 1.8.11')
+class TestWriteDirectChunk(TestCase):
+    def test_write_direct_chunk(self):
+
+        filename = "direct_chunk.hdf"
+        filehandle = h5py.File(filename, "w")
+
+        dataset = filehandle.create_dataset("data", (100, 100, 100), maxshape=(None, 100, 100), chunks=(1, 100, 100), dtype='float32')
+
+        # writing
+        array = numpy.zeros((10, 100, 100))
+        for index in range(10):
+            a = numpy.random.rand(100, 100).astype('float32')
+            dataset.id.write_direct_chunk((index, 0, 0), a.tobytes(), filter_mask=1)
```

Comment:
Double-check this on Python 2... I couldn't find a tobytes() method on the ndarray object under Python 2.7 with NumPy 1.8.2.